common/determined_common/experimental/trial.py

```diff
@@ -117,3 +117,6 @@ def select_checkpoint(
             best_checkpoint_func(r, key=lambda x: x["metrics"]["validation_metrics"][sort_by]),
             master=self._master,
         )
+
+    def __repr__(self) -> str:
+        return "Trial(id={})".format(self.id)
```

Comment:
I think this will be more useful if we print the hps here.

Reply:
Similar to the model. Were you thinking inline JSON?

Reply:
i'm in favor of pretty  print json

Reply:
Just realized this is the same situation as the `ExperimentReference`. We actually don't have the hparams in the object yet. Sorry for the confusion.